Src/ElasticScale.Client/Query/MultiShardDataReader.cs

```diff
@@ -1062,7 +1065,9 @@ public override bool Read()
                 //
                 if (this.PerformReadToFillBuffer())
                 {
-                    s_tracer.Verbose("MultiShardDataReader.Read.Complete; Duration: {0}; Shard: {1}", DateTime.UtcNow - readStartTime,
+                    stopwatch.Stop();
```

Comment:
Yeah, starting a stopwatch doesn't allocate anything  - it just grabs the value of `GetTimestamp()` and tracks it within the instance.  There's actually a lot of places here where `Stop()` didn't absolutely *need* to be called - but it's still better to do so.  It helps with debugging, and prevents mismatches if later more stuff is done than just the trace output.  (Similar defensive programming technique to how `if` statements should use curly braces even for a single line.)